### ISSUE TITLE
Segregated output SVG files

### ISSUE THREAD

**Issue by aravinth-r-7867** [evoactivity/ember-svg-jar#150]
How do I get more than one output SVG sprite files???

I want to store my final output SVG sprites based on different sections in my website. I don't want to load icons of one section in another section. We are using Ember engine technique, so each section loads as separate addon.

What I want to do is to separate my final output SVG sprite based on different sections. eg. svgs/admin, svg/support, svg/common, svg/utils.

```
svgJar: {
    strategy: 'symbol',

    symbol: {
      includeLoader: false,
      outputFile: '/assets/common/final.svg',
      outputFile: '/assets/admin/final.svg',
      outputFile: '/assets/support/final.svg',
    }
  }
```

#ember-addon #svg #ember-engine

**Comment by ivanvotti** [comment]
At this moment there's no way to do that in the addon.

**Comment by cannjeff** [comment]
Is this feature on the roadmap? Bundling all SVG's into the main vendor.js file while using engines isn't ideal.

**Comment by AravinthAro** [comment]
> Is this feature on the roadmap? Bundling all SVG's into the main vendor.js file while using engines isn't ideal.

Yes, it also affects caching of icons when the vendor.js file undergoes frequent changes.
